search-books/src/main/java/nix/education/entity/Book.java

```diff
@@ -0,0 +1,65 @@
+package nix.education.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import java.util.Date;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+import javax.persistence.Transient;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.apache.solr.client.solrj.beans.Field;
+
+@Entity
+@Data
+@Table(name = "books")
+@AllArgsConstructor
+@NoArgsConstructor
+public class Book {
+    @Id
+    @Field("id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Field("author")
+    private String author;
+
+    @Field("title")
+    private String title;
+
+    @Field("description")
+    private String description;
+
+    @ManyToMany(cascade = {
+        CascadeType.DETACH,
+        CascadeType.MERGE,
+        CascadeType.PERSIST,
+        CascadeType.REFRESH},
+        fetch = FetchType.LAZY)
+    @JoinTable(name = "book_tags",
+        joinColumns = {@JoinColumn(name = "book_id")},
+        inverseJoinColumns = {@JoinColumn(name = "tag_id")})
+    @JsonManagedReference(value = "tags")
```

Comment:
tags field does not serialize w/o this annotation?

Reply:
No, it works without this annotation.